cli/packages/cmd/login.go

```diff
@@ -252,6 +274,51 @@ func init() {
 	rootCmd.AddCommand(loginCmd)
 }
 
+func askForDomain() error {
+	//query user to choose between Infisical cloud or self hosting
+	options := []string{"Infisical Cloud", "Self Hosting"}
+	optionsPrompt := promptui.Select{
+		Label: "Select your hosting option",
+		Items: options,
+		Size:  2,
+	}
+
+	idx, _, err := optionsPrompt.Run()
```

Comment:
nit: is there a reason why you have choosen `idx`? I would think it's better to call it something like `selectedHostingOption`

Reply:
`idx` referring to the selected index of the options. However yes `selectedHostingOption` would be more readable. Instead of dealing with the index, comparing the selected value directly should be better. Thanks